README.md

```diff
@@ -369,4 +370,247 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
+```
+
+### 3. Increase Memory and CPUs
+
+In the Advanced tab of the Docker preferences, set 'CPUs' to 4.
+
+While Alfresco Digital Business Platform installs and runs with only 8 GiB allocated to Docker, 
+for better performance we recommend that 'Memory' value be set slightly higher, to at least 10 - 12 GiB
+(depending on the size of RAM in your workstation). 
+
+### 4. Change/Verify Context
+
+If you have previously deployed the DBP to AWS or minikube you will need to change/verify that the `docker-for-desktop` context is being used.
+
+```bash
+kubectl config current-context                 # Display the current context
+kubectl config use-context docker-for-desktop  # Set the default context if needed
+```
+
+### 5. Restart Docker  
+
+Docker can be faulty on its first start. So, it is always safer to restart it before proceeding. Right click on the Docker icon in the system tray, then left click "restart...". 
+
+### 6. Install Helm
+
+To install Helm, first you need to install Chocolatey using Command Prompt (with admin rights). 
+
+```bash
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+Install Helm
+
+```bash
+choco install kubernetes-helm
+```
+
+Initialize Tiller (Server Component)
+
+```bash
+helm init
+```
+
+### 7. Create your namespace
+
+```bash
+kubectl create namespace $DESIREDNAMESPACE
+```
+
+### 8. Pull secrets
+
+Log in to quay.io
+```bash
+docker login quay.io
+```
+Give username and password if prompted.
+
+In windows explorer, go to the current location of your command line.
+Create the file secrets.yaml, and past the following into it. 
+
+```bash
+apiVersion: v1
+kind: Secret
+metadata:
+  name: quay-registry-secret
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: replaceThis
+```
+Back in your command line, generate a base64 value for your dockercfg, this will allow Kubernetes to access quay.io
+
+```bash
+certutil -encode "%USERPROFILE%\.docker\config.json" tmp.b64 && findstr /v /c:- tmp.b64
+```
+From the output, copy the string located under:
+
+```bash
+CertUtil: -encode command completed successfully.
+```
+
+In the secrets.yaml file, replace 'replaceThis' with the copied string. 
+
+Note that when you paste the string in the data section, it may be pasted with new lines in it. So, make sure to take out the new lines. And leave a single space between "data:" and the string. 
+
+Now in your command line, remove the file that was just created as it is no longer needed. 
+
+```bash
+del tmp.b64
+```
+
+Create the secret in your namespace. 
+
+```bash
+kubectl create -f secrets.yaml --namespace $DESIREDNAMESPACE
+```
+
+### 9. Download the alfresco helm chart
```

Comment:
This doesn't actually download the chart, merely adds a remote chart repository to your helm configuration.

Reply:
changed to this:

### 9. Add remote chart repository to Helm configuration.

```bash
helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
```